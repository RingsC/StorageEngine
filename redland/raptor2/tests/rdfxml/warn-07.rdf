<?xml version="1.0" encoding="UTF-8"?> 
<rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
         xmlns="http://example.org/ns#">

  <!-- test warning for unknown rdf:parseType values (taken as Literal) -->

  <rdf:Description rdf:about="http://example.org/node">
    <prop1 rdf:parseType="Fake">
      <name>prop1name</name>
    </prop1>

    <!-- this is not a Collection but a bunch of XML -->
    <prop2 rdf:parseType="collection">
      <rdf:Description rdf:about="http://example.org/first" />
      <rdf:Description rdf:about="http://example.org/second" />
      <rdf:Description rdf:about="http://example.org/third" />
    </prop2>
  </rdf:Description>

</rdf:RDF>
