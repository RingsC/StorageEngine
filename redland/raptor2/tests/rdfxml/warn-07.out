<http://example.org/node> <http://example.org/ns#prop1> "\n      <name xmlns=\"http://example.org/ns#\">prop1name</name>\n    "^^<http://www.w3.org/1999/02/22-rdf-syntax-ns#XMLLiteral> .
<http://example.org/node> <http://example.org/ns#prop2> "\n      <rdf:Description xmlns:rdf=\"http://www.w3.org/1999/02/22-rdf-syntax-ns#\" rdf:about=\"http://example.org/first\"></rdf:Description>\n      <rdf:Description xmlns:rdf=\"http://www.w3.org/1999/02/22-rdf-syntax-ns#\" rdf:about=\"http://example.org/second\"></rdf:Description>\n      <rdf:Description xmlns:rdf=\"http://www.w3.org/1999/02/22-rdf-syntax-ns#\" rdf:about=\"http://example.org/third\"></rdf:Description>\n    "^^<http://www.w3.org/1999/02/22-rdf-syntax-ns#XMLLiteral> .
